Content.Shared/Turrets/TurretTargetSettingsSystem.cs

```diff
@@ -0,0 +1,78 @@
+using Content.Shared.Access;
+using Content.Shared.Access.Systems;
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared.Turrets;
+
+public sealed partial class TurretTargetSettingsSystem : EntitySystem
+{
+    [Dependency] private readonly AccessReaderSystem _accessReader = default!;
+
+    private ProtoId<AccessLevelPrototype> _accessLevelBorg = "Borg";
+    private ProtoId<AccessLevelPrototype> _accessLevelBasicSilicon = "BasicSilicon";
+
+    public override void Initialize()
+    {
+        base.Initialize();
+    }
+
+    public void SetAccessLevelExemption(Entity<TurretTargetSettingsComponent> ent, ProtoId<AccessLevelPrototype> exemption, bool enabled)
```

Comment:
Public API needs `[PublicAPI]` attribute and a doc-comment. If it is not a public API, mark this method private.
This applies to all other public methods in this ES.